Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -1774,9 +1778,23 @@ public JSONObject getFirstReferringParams() {
         return firstReferringParams;
     }
 
+    private CountDownLatch getFirstReferringParamsLatch = null;
```

Comment:
Can we move this member variable declaration to the variable declarations in the beginning  of the  class